client/pyproject.toml

```diff
@@ -4,6 +4,7 @@ description="AYON core addon."
 
 [tool.poetry.dependencies]
 python = ">=3.9.1,<3.10"
+markdown = "^3.4.1"
```

Comment:
Just to clarify, the dependency is already in AYON launcher, so it is not necessary to create new dependency package.